pex/version.py

```diff
@@ -1,4 +1,4 @@
 # Copyright 2015 Pex project contributors.
 # Licensed under the Apache License, Version 2.0 (see LICENSE).
 
-__version__ = "2.25.2"
+__version__ = "2.26.0"
```

Comment:
@jsirois I have prepared for a release here as well. Let me know if you want me to do any other changes.